ark/jsonschema/json.ts

```diff
@@ -0,0 +1,133 @@
+import {
+	printable,
+	throwParseError,
+	type array,
+	type ErrorMessage
+} from "@ark/util"
+import { type, type Out, type Type } from "arktype"
+import { parseJsonSchemaAnyKeywords } from "./any.ts"
+import { validateJsonSchemaArray, type inferJsonSchemaArray } from "./array.js"
```

Comment:
Have just noticed that some of these are using `.ts` and some are using `.js`. I'm not too familiar with ESM; which is correct?

Reply:
They should all be `.ts` now with the current build.

Reply:
Thanks; fixed in dd130e8ba